### ISSUE TITLE
Type misspelled "DEPSLIBS" in error message

### ISSUE THREAD

**Issue by glhenni** [TriBITSPub/TriBITS#295]
I got the following warning from a cmake invocation:

```
CMake Warning at /gpfs1/jenkins/serrano-slave/workspace/charon_serrano_nightly_opt/TriBITS/tribits/core/package_arch/TribitsLibraryMacros.cmake:690 (MESSAGE):
  WARNING: 'stk_util_registry' in DEPSLIBS is not a lib defined in the
  current cmake project! Such usage is deprecated (and will result in a
  configure error soon).  If this is an external lib you are trying to link
  in, it should likely be handled as a TriBITS TPL.  Otherwise, it should be
  passed in through IMPORTEDLIBS.  However, the only case we have found where
  IMPORTEDLIBS had to be used instead of through a proper TriBITS TPL is the
  C math library 'm'.
Call Stack (most recent call first):
  Trilinos/packages/stk/stk_util/stk_util/command_line/CMakeLists.txt:72 (TRIBITS_ADD_LIBRARY)
```

grepping for `DEPSLIBS` in the stk package yielded nothing because it should be `DEPLIBS`. (Note the extra *S* in the typo).

**Comment by bartlettroscoe** [comment]
Thanks for the FYI @glhenni!  I am in the process of pushing a fix to TriBITS 'master' right now and then I will post a Trilinos PR with an updated TriBITS snapshot.

**Comment by bartlettroscoe** [comment]
@glhenni,

FYI: This was fixed in TriBITS 'master' commit bef83cf and it is being updated in Trilinos 'develop' in PR https://github.com/trilinos/Trilinos/pull/5981.

**Comment by bartlettroscoe** [comment]
@glhenni 

FYI: This change was just merged to Trilinos 'develop' in the merged PR https://github.com/trilinos/Trilinos/pull/5981.

If you are good to go, then you can close this issue.

Thanks!
